### PR TITLE
Add “PiiLoggingEnabled” flag to MSALLogger

### LINKED ISSUE #190: Add "PiiLoggingEnabled" property to MSALLogger
In MSAL.NET PII logging isn't sent to the logging callback/delegate unless explicitly enabled, we should match that in MSAL ObjC

### PR DESCRIPTION
**Pull request by RPangrle**
This flag matches the one in MSAL.NET and should make it less likely an app developer inadvertantly logs PII due to ignorance.

Fixes #190

### CODE REVIEW

File: MSAL/src/MSALLogger.m [hunk 1]
```diff
@@ -59,6 +59,7 @@ - (id)init
     // will most likely be too noisy for most usage.
     self.level = MSALLogLevelInfo;
     self.consoleLogging = YES;
+    self.PiiLoggingEnabled = NO;
```
Comment (partnerinflight) [inline MSAL/src/MSALLogger.m]: nit: why reverse case, and not piiLogging?

Comment (RPangrle) [inline MSAL/src/MSALLogger.m]: Copying how it's done in .NET *shrug*. I've argued in the past that it's best for BOOL flags to default to false as well.

File: MSAL/test/unit/MSALLoggerTests.m [hunk 2]
```diff
@@ -78,27 +78,47 @@ - (void)testLogMacros
...
+    XCTAssertTrue([logger.lastMessage containsString:@"waiting on response from contoso.com"]);
+    XCTAssertEqual(logger.lastLevel, MSALLogLevelVerbose);
+}
+
+- (void)testIsPiiEnabled
+{
+    [[MSALLogger sharedLogger] setLevel:MSALLogLevelLast];
+    MSALTestLogger* logger = [MSALTestLogger sharedLogger];
+    [[MSALLogger sharedLogger] setPiiLoggingEnabled:NO];
+    LOG_VERBSOE_PII(nil, @"waiting on response from %@", @"contoso.com");
```
Comment (oldalton) [inline MSAL/test/unit/MSALLoggerTests.m]: Just noticed we have a funny typo here VERBSOE :)

Comment (RPangrle) [inline MSAL/test/unit/MSALLoggerTests.m]: oh wow. It's impressive we've had that typo for so long before someone noticed!

**Pull request merged**